examples/tfengine/multi_envs.hcl

```diff
@@ -120,14 +120,23 @@ template "cicd" {
     build_editors = ["group:example-cicd-editors@example.com"]
 
     terraform_root = "terraform"
+
+    # IMPORTANT: Cloud Source Repositories does not support code review or
+    # presubmit runs. If we set both plan and apply to run at the same time,
+	  # they will conflict and may error out. To get around this, for 'shared'
+	  # and 'prod' environment, set 'apply' trigger to not 'run_on_push',
+	  # and for other environments, do not specify the 'plan' trigger block
+	  # and let 'apply' trigger to 'run_on_push'.
     envs = [
       {
         name        = "shared"
         branch_name = "shared"
         triggers = {
           validate = {}
           plan = {}
-          apply = {}
+          apply = {
+            run_on_push = false # Do not auto run on push to CSR non-prod branch
```

Comment:
Add the trigger name to the comment as well like 

Do not auto run 'apply' trigger on push to CSR 'shared' branch

and similarly for other ones as well